SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc

```diff
@@ -1,25 +1,41 @@
-= Referenced Standards Requirements Conformance Support
+= References
 
-NOTE:  This file contains the general introduction to how SDPi handles standards conformance + integrates sections for each standard "ICS" statement. +
+// Appendix B
+[appendix#vol1_appendix_b_references,sdpi_offset=B]
+== References
+
+[%noheader]
+[%autowidth]
+[cols="1"]
+|===
+| *SDPi Supplement Version Note*: The inclusion of a References appendix is unique for IHE Technical Framework specifications.
+Typically, referenced standards sections are distributed throughout the specifications where appropriate; however, standards from other organizations (e.g., IEEE, ISO, IEC) typically have a "Normative References" clause and when desired a "Bibliography" clause.
+Also, integration of explicit standards' <<term_implementation_conformance_statement>> specifications (e.g., I<<ref_ieee_11073_10207_2017>> <<acronym_ics>> tables) is unique, but needed to support requirements interoperability.
 {empty} +
-ICS Statements per standard may be in a different subfolder and files - along with this appendix B file.
+Ultimately, the content of this appendix may be rearranged and even relocated; however, for early versions of the SDPi supplement, it has proven helpful, and even of critical importance and value.
+|===
 
-// Appendix B
-[appendix#vol1_appendix_b_referenced_standards_requirements_and_conformance,sdpi_offset=B]
-== Referenced Standards Requirements and Conformance
+// Appendix B.x
+[referenced_standards]
```

Comment:
This block type does not exist.
Please apply the following:

```suggestion
[bibliography]
```

as otherwise references are not rendered correctly.

Reply:
Update made to line 19.  Does this comment also mean to remove lines 8 & 9?

Reply:
No additional change required.